Lib/fontTools/varLib/__init__.py

```diff
@@ -208,6 +212,51 @@ def _add_avar(font, axes):
         assert +1.0 not in curve or curve[+1.0] == +1.0
         # curve.update({-1.0: -1.0, 0.0: 0.0, 1.0: 1.0})
 
+    if mappings:
+        interesting = True
+
+        derived = [
+            {
+                axes[name].tag: models.normalizeValue(v, vals_triples[axes[name].tag])
+                for name, v in mapping.inputLocation.items()
+            }
+            for mapping in mappings
+        ]
+        source = [
```

Comment:
i'm a bit confused by the names of the local variables `derived` and `source`, respectively associated with `inputLocation` and `outputLocation`.. (i'd have expected input being a "source" and output something "derived"). Could you clarify?

Reply:
I'll rename to `derived axes` and `source axes`, or just inputaxes and outputaxes.

Reply:
Ok but I still don't get our input is "derived" and output is "source"

Reply:
Think of a 'wght' axis that is derived from the parametric axes. The 'wght' is input and it drives the parametric axes. But it's really the derived axis.